service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -983,16 +970,7 @@ fun Database.Transaction.updateForm(
 
     // language=SQL
     val sql =
-        """
-WITH old_revisions AS (
-    UPDATE application_form SET latest = FALSE
-    WHERE application_id = :applicationId
-    RETURNING revision
-)
-INSERT INTO application_form (application_id, document, revision, latest)
-VALUES (:applicationId, :document, (SELECT coalesce(max(revision) + 1, 1) FROM old_revisions), TRUE)
-        """
-            .trimIndent()
+        "UPDATE application SET document = :document, form_modified = now() WHERE id = :applicationId;"
```

Comment:
We usually pass the current timestamp as a parameter from outside instead of using `now()` or similar database functions.

In this case using `now()` might be justified, though, because it's anyway used for `created` and `updated` timestamps, and there are ~5 places that would need to be changed.

Reply:
I think we *should* start passing these kind of things as parameters, especially in cases where the information is actually displayed to the user.

However, in this case `formModified` from the view doesn't seem to be used anywhere, so I'd just get rid of the whole column.

Reply:
I didn't want to remove `formModified` as the field was already a part of application view. However I added timestamp as parameter.

Reply:
removed `formModified`